packages/app/components/Settings/AddStudents.tsx

```diff
@@ -99,7 +147,30 @@ function RenderTable({
                   avatar={<Avatar src={item.photoURL} />}
                   title={item.name}
                 />
-                <div>{item.email}</div>
+                <Space>
+                  <div>{item.email}</div>
+                  {/* <Button
+                    type="primary"
+                    onClick={async (e) => {
+                      e.currentTarget.blur();
+                      const student = await API.profile.getUser(item.id);
+                      modifyFunction(student);
+                    }}
+                  >
+                    Edit
+                  </Button> */}
+                  <Button type="danger" onClick={() => showPopconfirm(item.id)}>
```

Comment:
We should follow convention and keep naming function in camelcase e.g. showPopConfirm